templates/pytorch/src/helpers/download_helpers.py

```diff
@@ -0,0 +1,100 @@
+import pickle
+import urllib
+import os
+from Algorithmia.errors import AlgorithmException
+from tempfile import mkdtemp
+
+def download_image(client, image_url, size=None):
+	"""
```

Comment:
These are also in tabs, while the other files are in 4 space indentation.

I'll make a commit to fix this in this PR too.